packages/platforms/delta/armel/arm-delta-ag6248c/arm-delta-ag6248c-poe/modules/builds/arm-delta-ag6248c-poe-cpld-mux-1.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * An I2C multiplexer dirver for delta as5812 CPLD
```

Comment:
Please update your comments. This was obviously sources from the Accton AS5812.

Reply:
65cd247a0acca7c14b7e320415da8f3665d73a69

Reply:
the comments is updated, please check it in the commit b1eeddcc3186b27c5066977cfd3687c061bca952